src/TwigRendererFactory.php

```diff
@@ -48,6 +52,9 @@ public function __invoke(ContainerInterface $container) : TwigRenderer
      * array having precedence.
      *
      * @param array|ArrayObject $config
+     *
+     * @return array
+     *
```

Comment:
Throughout this class, revert any new `@param` or `@return` annotations for values that are already typehinted and have no additional descriptions. Add descriptions for any `@throws` annotations to indicate when those scenarios occur.